app/uk/gov/hmrc/residencenilratebandcalculator/controllers/PurposeOfUseController.scala

```diff
@@ -26,6 +26,11 @@ import uk.gov.hmrc.residencenilratebandcalculator.{Constants, FrontendAppConfig,
 import uk.gov.hmrc.residencenilratebandcalculator.connectors.SessionConnector
 import uk.gov.hmrc.residencenilratebandcalculator.forms.PurposeOfUseForm
 import uk.gov.hmrc.residencenilratebandcalculator.views.html.purpose_of_use
+import play.api.libs.json.{Reads, Writes}
+import uk.gov.hmrc.residencenilratebandcalculator.models.UserAnswers
+import play.api.mvc._
+import uk.gov.hmrc.http.cache.client.CacheMap
+import uk.gov.hmrc.play.http.logging.SessionId
 
 @Singleton
 class PurposeOfUseController @Inject()(override val appConfig: FrontendAppConfig,
```

Comment:
Can we no longer inherit from SimpleControllerBase as we don't use any of its functionality anymore and so aren't really one.